verovio-util/verovio-proxy.js

```diff
@@ -4,200 +4,258 @@ verovio.vrvToolkit = verovio.vrvToolkit || {};
-	console.warn("Method renderPage is deprecated; use renderToSVG instead");
-	return verovio.vrvToolkit.renderToSVG(this.ptr, pageNo, JSON.stringify(options));
+verovio.toolkit.prototype.renderPage = function ( pageNo, options )
+{
+    console.warn( "Method renderPage is deprecated; use renderToSVG instead" );
+    return verovio.vrvToolkit.renderToSVG( this.ptr, pageNo, JSON.stringify( options ) );
 };
 
-verovio.toolkit.prototype.renderToMIDI = function (options) {
-	return verovio.vrvToolkit.renderToMIDI(this.ptr, JSON.stringify(options));
+verovio.toolkit.prototype.renderToMIDI = function ( options )
+{
+    return verovio.vrvToolkit.renderToMIDI( this.ptr, JSON.stringify( options ) );
 };
 
-verovio.toolkit.prototype.renderToMidi = function (options) {
-	console.warn("Method renderToMidi is deprecated; use renderToMIDI instead");
-	return verovio.vrvToolkit.renderToMIDI(this.ptr, JSON.stringify(options));
+verovio.toolkit.prototype.renderToMidi = function ( options )
+{
+    console.warn( "Method renderToMidi is deprecated; use renderToMIDI instead" );
+    return verovio.vrvToolkit.renderToMIDI( this.ptr, JSON.stringify( options ) );
 };
 
-verovio.toolkit.prototype.renderToSVG = function (pageNo, options) {
-	return verovio.vrvToolkit.renderToSVG(this.ptr, pageNo, JSON.stringify(options));
+verovio.toolkit.prototype.renderToSVG = function ( pageNo, options )
+{
+    return verovio.vrvToolkit.renderToSVG( this.ptr, pageNo, JSON.stringify( options ) );
 };
 
-verovio.toolkit.prototype.renderToTimemap = function () {
-	return JSON.parse(verovio.vrvToolkit.renderToTimemap(this.ptr));
+verovio.toolkit.prototype.renderToTimemap = function ()
+{
+    return JSON.parse( verovio.vrvToolkit.renderToTimemap( this.ptr ) );
 };
 
-verovio.toolkit.prototype.setOptions = function (options) {
-	verovio.vrvToolkit.setOptions(this.ptr, JSON.stringify(options));
+verovio.toolkit.prototype.setOptions = function ( options )
+{
+    verovio.vrvToolkit.setOptions( this.ptr, JSON.stringify( options ) );
 };
 
 /***************************************************************************************************************************/
 
 // If the window object is defined (if we are not within a WebWorker)...
-if ((typeof window !== "undefined") && (window.addEventListener)) {
-	// Add a listener that will delete the object (if necessary) when the page is closed
-	window.addEventListener("unload", function () {
-		for (var i = 0; i < verovio.instances.length; i++) {
-			verovio.vrvToolkit.destructor(verovio.instances[i]);
-		}
-	});
+if ( ( typeof window !== "undefined" ) && ( window.addEventListener ) )
+{
+    // Add a listener that will delete the object (if necessary) when the page is closed
+    window.addEventListener( "unload", function ()
+    {
+        for ( var i = 0; i < verovio.instances.length; i++ )
+        {
+            verovio.vrvToolkit.destructor( verovio.instances[i] );
+        }
+    } );
 }
```

Comment:
Very minor nitpick, but I'm wondering if the formatter ran on this file only? Most of the changes are white space only but the formatting seems different than the other files, namely the ``{`` on its own line

Reply:
This is taken directly from verovio so I'm actually not sure what happened here.